src/common/dom/apply_themes_on_element.ts

```diff
@@ -29,22 +29,34 @@ export const applyThemesOnElement = (
   localTheme,
   mainElement = false
```

Comment:
btw having line 94 in this function is sooo weird. That should be in the theme mixin or something ?